pkg/bootstrap/node.go

```diff
@@ -329,60 +331,55 @@ func runModprobe(moduleName string) error {
 	return nil
 }
 
-func ensureOverlayfs() {
-	if isOverlayfsAvailable() {
-		return
-	}
+func ensureOverlayfs() error {
+	if !isOverlayfsAvailable() {
 
-	log.Println("Warning: overlayfs not found, docker would not run.")
-	log.Println("loading overlay module... ")
+		// This is an incorrect assumption. It depends on the docker
+		// storage driver
```

Comment:
In this function, code looks effectively similar as before.
But I'm not sure I understand this comment.
Are you saying that we should check for docker storage driver in the future?
In that case, we should create another github issue, or add `TODO` to the comment.

Reply:
issue added
https://github.com/kinvolk/kube-spawn/issues/302

Reply:
Thanks for adding the issue.